benchopt/tests/__init__.py

```diff
@@ -59,7 +59,8 @@ def __exit__(self, exc_class, value, traceback):
                 stem = path_.stem
                 html_files = dir_.glob('*' + stem + '*.html')
                 for html_file in html_files:
-                    html_file.unlink(missing_ok=True)  # remove html files
+                    if html_file.exists():
+                        html_file.unlink()  # remove html files
```

Comment:
How can it not exists as it is given by `glob`?

Reply:
True.... I'll remove this